part 2 answers via pico

```diff
@@ -0,0 +1,8 @@
+I used linux commands such as cd , ls, and others for traversing the VM
```

Comment:
This was meant to be a reference for commands where you list each command and what it does